PyISY/Nodes/node.py

```diff
@@ -69,9 +69,9 @@ def parse_xml_properties(xmldoc):
             #print "prop=",prop.toprettyxml();
             units = uom if uom == 'n/a' else uom.split('/')
             if (val == ""):
-                val = 0
+                val = -1
             else:
-                val = int(val.replace(' ', '0'))
+                val = int(val.replace(' ', '-1'))
```

Comment:
This line would cause problems if the value was '  5'. I'm not sure if that every happens, but it feels plausible. I really don't like any of the original logic on these lines. What if we did something like this:

```python
val = val.strip()
if val == "":
    val = -1
val = int(val)
```